dataline-singer/src/main/resources/json/SingerMessage.json

```diff
@@ -4,13 +4,13 @@
   "title": "SingerMessage",
   "description": "This is a compromise. Java codegen doesn't support union types. So we need to just make a flat json schema that looks like at of the possible permutations of a singer message. See the description for each field to see which message type that field belongs to.",
   "type": "object",
-  "additionalProperties": false,
+  "additionalProperties": true,
   "required": ["type"],
   "properties": {
     "type": {
       "description": "record, schema, and state message: used to determine record type. used in all singer messages.",
       "type": "string",
-      "enum": ["RECORD", "SCHEMA", "STATE"]
+      "enum": ["RECORD", "SCHEMA", "STATE", "ACTIVATE_VERSION"]
```

Comment:
some context around `ACTIVATE_VERSION`: 
- confused person [on singer slack](https://singer-io.slack.com/archives/C0109UR8G73/p1595950100423300)
- some comments in the source code of a random tap explaining it: [link](https://github.com/ramtribe/Scripts/blob/46f5b15996cb4a29217b41a630e199d871a89280/facebook_business/Lib/site-packages/singer/messages.py#L136)
- i opened an issue in Singer asking to document it [link](https://github.com/singer-io/getting-started/issues/71)

Reply:
Can we hold off on implementing it? or does it break smth to not have it?

Reply:
I don't think it should break anything since this isn't an officially supported field yet.

Reply:
woof. yeah. i'd rather just ignore this type of message then add it into "our" protocol.